src/common.rs

```diff
@@ -528,6 +528,8 @@ pub enum DiskType {
 /// [`SystemExt::SUPPORTED_SIGNALS`][crate::SystemExt::SUPPORTED_SIGNALS].
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Debug)]
 pub enum Signal {
+    /// Null to check if process is still alive.
+    Null,
```

Comment:
This is a public API so better not put it here (but this is the right signal to send to check if a process is alive indeed!).

Reply:
Just like that is fine.